examples/typescript-example/src/App.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { FC, useEffect, useRef } from 'react'
+import {
+  useTable,
+  useGroupBy,
+  useFilters,
+  useSortBy,
+  useExpanded,
+  usePagination,
```

Comment:
Are you going to show the example of using usePagination soon?

Reply:
I can try but bear in mind that this example is not the official way and is just a way to circumnavigate types doing some cast, so I'm not sure you want to go this way

Reply:
yeah, I'm fine going this way. The sooner you can show using usePagination, the better, so i'll get unblocked. Thanks!

Reply:
what's the issue you are having, did you try to build it yourself with pagination?

Reply:
Haven't tried yet. I can't find the tutorial to do it.

Reply:
Check the example on the main readme and try to integrate that part with this file above it might just work!

Reply:
again is not the prettiest approach since we have to cast in more and more places so I'm not sure as the other are saying this is a good approach but if you want to go this way this is how it should work

Reply:
Thanks! Will look at it.